src/structs/cell.rs

```diff
@@ -201,23 +201,20 @@ impl Cell {
     }
 
     pub(crate) fn get_width_point_cell(&self) -> f64 {
-        let mut max_point = 0f64;
         let value = self.get_formatted_value();
-        let value_list: Vec<&str> = value.split('\n').collect();
-        for value in value_list {
+
+        value.split('\n').fold(0f64, |mut acc, value| {
             let mut point = 0f64;
             for chr in value.chars() {
-                let mut clen = chr.len_utf8() as f64;
-                if clen > 1f64 {
-                    clen = 1.5;
-                }
+                let clen = if chr.len_utf8() > 1 { 1.5 } else { 1.0 };
```

Comment:
here if `clen` if checked for greater than 1 then as `len_utf8()` can give only values 1to4 inclusive then this logic is similar to previous.